### PR TITLE
Add new feature - Send tweet using Twitter Api

### PR DESCRIPTION
**Pull request by sk364**
Usage :

- Create a Twitter App
- Fetch Consumer Key, Consumer Secret, Access Token Key, Access Token Secret
- Initialize the bot with the fetched keys in the step above
- `bot.send_tweet(tweet_text)` will post a tweet

Simple Use Case :
Automate tweeting.
`>> tweet_data =  open('tweet_texts').read()`
`>> replies = [bot.send_tweet(tweet) for tweet in tweet_data]`

### CODE REVIEW

File: bot_chucky/bot.py [hunk 1]
```diff
@@ -54,8 +68,20 @@ def send_weather_message(self, _id, city_name):
...
+
+    def send_tweet(self, status):
+        """
+        :param status: Tweet text, type -> str
+        """
+        if not all(self.twitter_tokens.values()):
+            raise BotChuckyTokenError('Twitter')
+
+        reply = self.twitter.send_tweet(status)
+
```
Comment (MichaelYusko) [inline bot_chucky/bot.py]: Would be coo, l if tweet sent successfully, bot will return message to the user something like `I'm sent your tweet`, or you can propose your option how message should look like;)

Comment (MichaelYusko) [inline bot_chucky/bot.py]: And other things looks, nice.

Comment (sk364) [inline bot_chucky/bot.py]: Sure. I was wondering what to reply. I will update the message.

Comment (MichaelYusko) [inline bot_chucky/bot.py]: @sk364 I'll refactor your code a little bit later, but everything looks good 👍 
If you will have a free time, check tickets please.

https://github.com/MichaelYusko/Bot-Chucky/issues/27
https://github.com/MichaelYusko/Bot-Chucky/issues/15

**Pull request merged**